Added a book and a video course

poochingil opened the pull request:
Book - Android Development
Youtube Channel - The Complete Android Development Course

androiddevnotes reviewed (changes requested):
Thank you for the Pull Request!

The amazon link gives a 404 error.
The YouTube channel does not have up to date and promising content for Android Kotlin.

Please join the Discord server where we discuss open-source, android dev, and content creation. Invite link - https://discord.gg/vBnEhuC

poochingil commented:
> Thank you for the Pull Request!
> 
> The amazon link gives a 404 error.
> The YouTube channel does not have up to date and promising content for Android Kotlin.
> 
> Please join the Discord server where we discuss open-source, android dev, and content creation. Invite link - https://discord.gg/vBnEhuC

Added Book -> Android Cookbook
Added Video Course -> The Complete Android N Developer Course
Removed Youtube Channel -> Android Developer
